cmd/lncli/cmd_state.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/urfave/cli"
+)
+
+var getStateCommand = cli.Command{
+	Name:     "state",
+	Category: "Startup",
+	Usage:    "Get the current state of the wallet and RPC",
+	Description: `
+	Get the current state of the wallet. The possible states are:
+	 NON_EXISTING: wallet has not yet been initialized.
```

Comment:
nit: prefix each item with `-` to give it more of a list/enumeration look and feel?

Reply:
Yep!